### ISSUE TITLE
Scheduling Specific DateTimes

### ISSUE THREAD

**Issue by Lachlan24** [rafaelsetragni/awesome_notifications#4]
Hey, is it possible to schedule 2 or more Specific dateTimes with cron expression. For example how might i write these two UTC times.

2020-09-16 19:34:11.448002Z
2020-09-17 22:57:11.449339Z

Thanks

**Comment by rafaelsetragni** [comment]
OBS: Remembering that the maximum precision is determinated in **seconds** to schedule a notification.

Lets considered the examples bellow:

2020-09-16 19:34:11.000000Z
2020-09-17 22:57:11.000000Z

So, to schedule the notifications on those respective times:


```dart
to: "[APP_TOKEN]",
collapse_key: "type_a",
data: {
	content: {
		id: 158476,
		channelKey: "shedules_channel",
		title: "A schedule example",
		body: "This is a notification scheduled to play two times",
		payload: {
			"secret_id":"00001"
		}
	},
	schedule: {
		initialDateTime: "2020-09-16 19:34:11",
		crontabSchedule: "11 57 22 ? 17 09 2020"
	}
}
```

**Comment by Lachlan24** [comment]
And if i wanted to add a 3rd, for example: 2020-09-19 21:27:11 What would be the pattern then? Thanks

**Comment by rafaelsetragni** [comment]
The cron expression is limited and is not possible to schedule so much dates with such precision.

But i guess is possible to put a extra configuration field on schedule parameter to receive a list of dates, as the example bellow:

```dart
to: "[APP_TOKEN]",
data: {
	content: {
		id: 158476,
		channelKey: "shedules_channel",
		title: "A schedule example",
		body: "This is a notification scheduled to play two times",
		payload: {
			"secret_id":"00001"
		}
	},
	schedule: {
		initialDateTime: "2020-09-16 19:34:11",
		preciseSchedules: [
			"2020-09-16 19:34:11"
			"2020-09-17 22:57:11",
			"2020-09-18 14:37:11",
			"2020-09-19 21:27:11",
			"2020-09-20 12:57:11"
		}
	}
}
```

And even `initialDateTime` turns unnecessary in this example.

I could include that new feture in the next update. What you think?

**Comment by Lachlan24** [comment]
Yes please, that would be greatly appreciated. Thankyou for all your help.

**Comment by rafaelsetragni** [comment]
I released a new version (0.0.2), containing the `preciseSchedules` field, to do what you need.

I haven't tested this new feature with a unit test case, but I will be doing it soon. I just need to finish the iOS side before, because now the progress of the platforms is very disproportionate to each other.
